generator/internal/client_cc_generator.cc

```diff
@@ -56,9 +64,21 @@ bool GenerateClientCC(pb::ServiceDescriptor const* service,
     p->Print("\nnamespace $namespace$ {", "namespace", nspace);
   }
 
-  p->Print(vars,
-           "\n"
-           "// TODO: add content");
+  DataModel::PrintMethods(service, vars, p, R"method(
```

Comment:
My thought process was almost the exact opposite w.r.t. formatting the output and code review :P
The line of reasoning was not formatting at all would reduce the risk of extraneous whitespace changes in PRs. It's clearly a bad idea to add an additional dependency to a formatter, but it seemed likely that anyone using the generator in a workflow more serious than an exploratory one would be fine adding that additional step themselves.

Reply:
Heh :P
Is there a danger of whitespace changes in PRs? I agree they may occur, but what is the danger exactly? Just that they will be troublesome? I feel pretty strongly that we need to be able to do a sensible review of the proposed code changes, and that that just isn't possible without formatting the generated code.

Re exploratory users - I agree with your point, but one of the things that an exploratory user will very likely want to do is generate and then *read* the generated code, and we really aren't making that very easy.

Reply:
Since clang-format is part of the CI pipeline I suppose the risk of extraneous whitespace changes is greatly minimized. Just a defensive habit on my part from other projects. Will add some commonsense formatting to the new templates.